pkg/jobs/jobspb/jobs.proto

```diff
@@ -149,6 +149,9 @@ message StreamIngestionProgress {
   // back any additional ingested data, to bring the ingested cluster to a
   // consistent state as of the CutoverTime.
   util.hlc.Timestamp cutover_time = 1 [(gogoproto.nullable) = false];
+  // CutoverStarted is set by the stream ingestion job when the cutover has
+  // started.
+  bool cutover_started = 6;
```

Comment:
right, that's the hydrated state we discussed, it's on my todo list and I kept pushing it because I was thinking the current code is simple and fine.. BUT.. you and Aditya both asked for it then here it is.. added in a separate a commit, PTAL.